css/filesbutton.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * @copyright Copyright (c) 2020 cnmicha <cnmicha@bhb-networks.com>
+ *
+ * @author cnmicha <cnmicha@bhb-networks.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#open-in-photos-button {
+	background-color: transparent;
+	border: none;
+	margin: 0;
+	padding: 22px;
+	opacity: 0.5;
+	position: fixed;
+	right: 44px;
+	z-index: 100;
+}
+
+#open-in-photos-button:hover {
+	opacity: 1
+}
```

Comment:
If you use a vue component instead of a js file, you can bundle the css in it directly if you want :)

Reply:
I thought it would be too much overhead to create a second (separate) vue instance in the files app. But anyways, I think there is no other possibility to add the NC tooltip styling without using vue or creating duplicate code, so I'll use vue. Thanks :)

Reply:
Resolved in https://github.com/cnmicha/photos/commit/1091cdee3688d1f4d5a1dda01eeb7cae29f00653